### ISSUE TITLE
MAM5 and updates to dust emis

### ISSUE THREAD

**Issue by fvitt** [ESCOMP/CAM#141]
Add a mode for stratospheric sulfate to the modal aerosol model.
Updates to dust emissions.

Francis, this is my F90 codes and BLD codes to setup MAM5 configuration

/glade/u/home/zke3/scratch/cases/edit8

The F90 codes and chem_mech.in are in CAM, configure codes are in BLD folder, chem_mech.in and namelist files for WACCM are in WACCM folder. The F90 files are used for both WACCM and CAM. Two configurations I added in is trop_mam5 and waccm_tsmlt_mam5. If you need the codes to separate with the convection changes, I also have that older version. That is based on “release-cesm2.1.2” version.

Thanks 

Ziming Ke

Francis, here is our code changes,

/glade/u/home/longei/CAM-MAM5-codes/wet_removal_n02_cam6_2_020

 

and a list of the routines we modified.

DUSTMod.F90

dust_model.F90

modal_aer_opt.F90

Thanks very much.

All the best & Be well,

Longlei

**Comment by fvitt** [comment]
Not required for cesm2.2.  However it is desired get the changes into cam_development soon after code freeze.

**Comment by fvitt** [comment]
Email from Longlei (ll859@cornell.edu):

Below is the directory where the source code is recorded. 

/glade/work/longei/Cheyenne/TESTS/NCAR2dps/CAM6_2dps

 

As said before, I made the old dust emission scheme optional. I defined a new namelist variable named as brift_dps, where brift stands for BRIttle Fragmentation Theory, and dps stands for Dust Production Scheme.

 

By default, the model will automatically use the new dust emission scheme. If one wants to go back the old dust scheme, he/she should set the following in BOTH user_nl_cam and user_nl_clm.

 

brift_dps = .true.
